### PR TITLE
Feature/aoli fancyfooterwork

### PR DESCRIPTION
**Pull request by Aoliveras**
Needed to push one more minimal change before PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,9 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
+<<<<<<< HEAD
+.vscode
+=======
```
Comment (avalojc) [inline .gitignore]: looks like a merge conflict

**Review by avalojc — CHANGES_REQUESTED** [review]
code looks good but there is a merge conflict in the git ignore

File: .gitignore (hunk 1 above)
Comment (BrianLoveGa) [reply .gitignore]: i added and changed that so the vs code settings would stay out of the repo.... gonna squash and merge it in

**Review by BrianLoveGa — APPROVED** [review]
good to go

**Review by avalojc — APPROVED** [review]
approve per brian

File: .gitignore [hunk 2]
```diff
@@ -25,9 +25,6 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
```
Comment (mrterranova) [inline .gitignore:27]: Edited the file

**Review by avalojc — APPROVED** [review]

**Pull request merged**